projects/storefrontlib/cms-components/product/product-list/product-facet-navigation/facet-list/facet-list.component.ts

```diff
@@ -109,4 +130,55 @@ export class FacetListComponent {
   block(event?: MouseEvent) {
     event?.stopPropagation();
   }
+
+  protected subscribeToFocusUpdates(): void {
+    const updateFocusSub = this.facetService.facetList$.subscribe(
+      (facetList) => {
+        // TODO: (CXSPA-7321) - Remove feature flag next major release
+        const isFeatureEnabled = this.featureConfigService?.isEnabled(
+          'a11yFacetsDialogFocusHandling'
+        );
+        if (!isFeatureEnabled) {
+          return;
+        }
+        this.handleDialogFocus(facetList.facets);
+      }
+    );
+    this.subscriptions.add(updateFocusSub);
```

Comment:
Now we can make this method simpler.

```suggestion
    this.subscriptions.add(
      this.facetService.facetList$.subscribe(
        (facetList) =>
          this.handleDialogFocus(facetList.facets);
      )
    );
```